feature/_fish/install.sh

```diff
@@ -1,7 +1,7 @@
 . /feature-installer-utils.sh
 
 FLAVOUR="X$(cat /home/cloudcontrol/flavour)X"
-if [ "${FLAVOUR}" == "XazureX" ] || [ "${FLAVOUR}" == "XsimpleX" ] || [ "${FLAVOUR}" == "XtanzuX" ]
+if [ "${FLAVOUR}" == "XazureX" ] || [ "${FLAVOUR}" == "XsimpleX" ] || [ "${FLAVOUR}" == "XtanzuX" ] || [ "${FLAVOUR}" == "XgcloudX" ]
```

Comment:
if it's a bash, how about 
`if [ ${FLAVOUR} =~ X(azure|simple|tanzu|gcloud)X ]`

Reply:
Good idea!